.circleci/config.yml

```
@@ -101,6 +155,57 @@ jobs:
 
+  safety-scan:
+    executor: python-simple
+    steps:
+      - checkout
+      - run:
+          name: Create virtual environment and install dependencies
+          command: |
+            python3 -m venv /tmp/safety-scan-env
+            source /tmp/safety-scan-env/bin/activate
+            pip install --upgrade pip
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)